tex/chProgramming.tex

```diff
@@ -2220,9 +2217,9 @@ \section{Exercises}
 
 \begin{coq}{}{width=8cm}
 Notation "( A , B , C )" := (triple A B C).
-Notation "p .1" := (fst p).
-Notation "p .2" := (snd p).
-Notation "p .3" := (thrd p).
+Notation "p .1" := (fst p) (at level 5).
```

Comment:
Wouldn't it be better to use level 2, as the library does for an analogous datatype? IMHO, it can prevent confusion in a notation-heavy development.